app/src/main/java/uc/edu/ozturkkl/traveltracker/ui/main/MainFragment.kt

```diff
@@ -6,7 +6,9 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import uc.edu.ozturkkl.traveltracker.R
+import kotlinx.android.synthetic.main.main_fragment.*
```

Comment:
Synthetic import for the screen layout to use the button.

Reply:
Synthetic imports: very good; save a lot of typing.